Packs/Sysdig/Integrations/Sysdig/README.md

```diff
@@ -0,0 +1,215 @@
+Security for containers, Kubernetes, and cloud.
+## Configure Sysdig on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Sysdig.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Required** |
+    | --- | --- |
+    | Server URL (e.g., https://us2.app.sysdig.com) | True |
+    | Sysdig Monitor API Key | True |
+    | Sysdig Secure API Key | True |
+    | Trust any certificate (not secure) | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### sysdig-get-alerts
+***
+Returns existing alerts
```

Comment:
add the outputs of the following commands and then update/generate the docs accordingly:
- sysdig-get-alerts 
- sysdig-get-metrics

Also it will be very helpful if you will generate the docs with examples, see this article:
https://xsoar.pan.dev/docs/documentation/readme_file#command-examples---integration-documentation

If you need assistance, please let me know.
I have a user in slack DFIR